frontend/src/component/segments/SegmentForm.tsx

```diff
@@ -12,9 +12,11 @@ export type SegmentFormMode = 'create' | 'edit';
 interface ISegmentProps {
     name: string;
     description: string;
+    project: string;
```

Comment:
Should this be 
```suggestion
    project: string | undefined;
```
?